datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py

```diff
@@ -1,21 +1,79 @@
+import glob
+import os
+
 import click
 
-from .....fs import dir_exists, file_exists
-from ...console import CONTEXT_SETTINGS, abort, echo_failure
+from datadog_checks.dev.tooling.constants import get_root
+
+from ...console import CONTEXT_SETTINGS, echo_failure, echo_info, echo_success, echo_warning
 from . import validators
+from .validators.utils import exist_profile_in_path, initialize_path
+
+MESSAGE_METHODS = {'success': echo_success, 'warning': echo_warning, 'failure': echo_failure, 'info': echo_info}
 
 
 @click.command("validate-profile", short_help="Validate SNMP profiles", context_settings=CONTEXT_SETTINGS)
 @click.option('-f', '--file', help="Path to a profile file to validate")
 @click.option('-d', '--directory', help="Path to a directory of profiles to validate")
 @click.option('-v', '--verbose', help="Increase verbosity of error messages", is_flag=True)
 def validate_profile(file, directory, verbose):
+    path = initialize_path(directory)
+
     if file:
-        if not file_exists(file):
-            echo_failure("Profile file not found, or could not be read: " + file)
-            abort()
-    if directory:
-        if not dir_exists(directory):
-            echo_failure("Directory not found, or could not be read: " + directory)
-            abort()
-    validators.validate_profile(file, directory, verbose)
+        _validate_single_profile(file, path, verbose)
+
+    else:
+        if not directory:
+            dd_profiles_path = os.path.join("snmp", "datadog_checks", "snmp", "data", "profiles")
+            directory = os.path.join(get_root(), dd_profiles_path)
```

Comment:
It seems we have a duplication of logic for:

```
join(get_root(), 'snmp', 'datadog_checks', 'snmp', 'data', 'profiles')
```

That is already present in 


```
def initialize_path(directory):
    path = []
    path.append('./')

    path.append(join(get_root(), 'snmp', 'datadog_checks', 'snmp', 'data', 'profiles'))
```

I think it can help to have a function like `get_default_snmp_profiles_path` like and then use it:

```
def get_default_snmp_profiles_path(directory):
    return join(get_root(), 'snmp', 'datadog_checks', 'snmp', 'data', 'profiles')
```